tests/linearizability/linearizability_test.go

```diff
@@ -192,3 +185,39 @@ type trafficConfig struct {
 	clientCount int
 	traffic     Traffic
 }
+
+func checkOperationsAndPersistResults(t *testing.T, operations []porcupine.Operation, clus *e2e.EtcdProcessCluster) porcupine.CheckResult {
+	path, err := filepath.Abs(filepath.Join(resultsDirectory, strings.Replace(t.Name(), "/", "_", -1)))
+	if err != nil {
+		t.Error(err)
+	}
+	err = os.MkdirAll(path, 0700)
+	if err != nil {
+		t.Error(err)
+	}
+
+	linearizable, info := porcupine.CheckOperationsVerbose(etcdModel, operations, 0)
+	if linearizable != porcupine.Ok {
+		for _, member := range clus.Procs {
+			memberDataDir := filepath.Join(path, member.Config().Name)
+			err := os.RemoveAll(memberDataDir)
+			if err != nil {
+				t.Error(err)
+			}
+			t.Logf("saving %s data dir to %q", member.Config().Name, memberDataDir)
+			err = os.Rename(member.Config().DataDirPath, memberDataDir)
+			if err != nil {
+				t.Error(err)
+			}
+		}
+	}
+	t.TempDir()
```

Comment:
?
```suggestion

```

Reply:
You resolved this comment without providing any explanation. Why do you add this?